### PR TITLE
Samples now work on arm and arm64 K8s.

### PR DESCRIPTION
**Pull request by artursouza**
Publishes arm and arm64 images.

# Description

Fixes Dockerfiles to work on arm and arm64.

Validation build: https://github.com/artursouza/quickstarts/actions/runs/400197264

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #327 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] The quickstart code compiles correctly
* [X] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary

### CODE REVIEW

File: distributed-calculator/csharp/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,18 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build-env
+FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim-amd64 AS build-env
```
Comment (artursouza) [inline distributed-calculator/csharp/Dockerfile:1]: Build step for dotnet core must be on amd64 and the platform is set at the restore command.

**Review by youngbupark — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -7,28 +12,54 @@ on:
...
       - v*
+    paths-ignore: 
+      - '**.md'
   pull_request:
     branches:
       - master
       - release-*
-
+    paths-ignore: 
+      - '**.md'
```
Comment (youngbupark) [inline .github/workflows/build.yml:22]: this will block merge only if md files are changed. because GH checks will ensure all sample builds are done.

Comment (artursouza) [reply .github/workflows/build.yml:22]: OK. Changing it.

**Review by youngbupark — APPROVED** [review]

**Comment by artursouza** [comment]
I know why it gets stuck. It is because the build names "Build and push docker images" does not exist anymore. Please, force merge this.

**Comment by youngbupark** [comment]
> I know why it gets stuck. It is because the build names "Build and push docker images" does not exist anymore. Please, force merge this.

I changed the branch checkbuild requirement. you are good to merge it now.

**Pull request merged**